src/core/utils/bounding-box.ts

```diff
@@ -189,10 +189,10 @@ function traverseAndCollectTrackInfo(
 
         if (spec.orientation === 'vertical') {
             // This is a vertical view, so use the largest `height` of the tracks for this view.
-            cumHeight = Math.max(...tracks.map(d => d.height));
+            cumHeight = Math.max(...tracks.map(d => d.height!));
             tracks.forEach((track, i, array) => {
                 if (getNumOfXAxes([track]) === 1) {
-                    track.width += HIGLASS_AXIS_SIZE;
+                    track.width! += HIGLASS_AXIS_SIZE;
```

Comment:
Same here. If these are truly defined, then our type system should show that somehow.